ironfish/src/rpc/routes/wallet/postTransaction.test.ts

```diff
@@ -2,23 +2,29 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-import { v4 as uuid } from 'uuid'
 import { RawTransactionSerde } from '../../../primitives/rawTransaction'
+import { useAccountFixture } from '../../../testUtilities'
 import { createRawTransaction } from '../../../testUtilities/helpers/transaction'
 import { createRouteTest } from '../../../testUtilities/routeTest'
+import { Account } from '../../../wallet/account'
 
 describe('Route wallet/postTransaction', () => {
   const routeTest = createRouteTest(true)
+  let account: Account
+
+  beforeAll(async () => {
+    account = await useAccountFixture(routeTest.node.wallet, 'existingAccount')
```

Comment:
It's preferable not to use fixtures in beforeAll, these fixtures get generated for every test weather they use them or not when more tests are added in the future. You can just use it in each test.